gtk-3.22/gtk-widgets.css

```diff
@@ -1783,13 +1793,12 @@ scrollbar.overlay-indicator:hover trough {
     background-color: alpha (#000, 0.1);
 }
 
-scrollbar:hover trough {
-    background-color: transparent;
-    border-radius: 0;
-}
-
-trough {
-    background-color: @base_color;
+/* FIXME: background is ok, but nothing seems affecting junction border.
+ * Even placing this block at the end of file doesn't help.
+ * Can't find source, unless inspector will show junction node */
+scrolledwindow > junction, {
```

Comment:
because of the `,` this isn't a valid selector. You probably also don't need the `>` here since we can probably assume there won't be a junction inside of a junction

Reply:
yeah, even adwaita css has it without '>' .